pkgs/servers/http/nginx/generic.nix

```diff
@@ -79,11 +79,11 @@ stdenv.mkDerivation {
     "--http-log-path=/var/log/nginx/access.log"
     "--error-log-path=/var/log/nginx/error.log"
     "--pid-path=/var/log/nginx/nginx.pid"
-    "--http-client-body-temp-path=/var/cache/nginx/client_body"
-    "--http-proxy-temp-path=/var/cache/nginx/proxy"
-    "--http-fastcgi-temp-path=/var/cache/nginx/fastcgi"
-    "--http-uwsgi-temp-path=/var/cache/nginx/uwsgi"
-    "--http-scgi-temp-path=/var/cache/nginx/scgi"
+    "--http-client-body-temp-path=/tmp/nginx_client_body"
+    "--http-proxy-temp-path=/tmp/nginx_proxy"
+    "--http-fastcgi-temp-path=/tmp/nginx_fastcgi"
+    "--http-uwsgi-temp-path=/tmp/nginx_uwsgi"
+    "--http-scgi-temp-path=/tmp/nginx_scgi"
```

Comment:
Is this path sandboxed in our systemd unit?

Reply:
Yes.

Reply:
This change breaks my workflow™
I'm serving temporary files from `/tmp/data` and have `systemd.services.nginx.serviceConfig.PrivateTmp = mkForce false;` to make this work. After this change nginx fails to start:
```
Dec 06 12:37:03 nixos nginx-pre-start[2881]: nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
Dec 06 12:37:03 nixos nginx-pre-start[2881]: nginx: [emerg] mkdir() "/tmp/nginx_client_body" failed (30: Read-only file system)
Dec 06 12:37:03 nixos nginx-pre-start[2881]: nginx: configuration file /etc/nginx/nginx.conf test failed
Dec 06 12:37:03 nixos systemd[1]: nginx.service: Control process exited, code=exited, status=1/FAILURE
```

Reply:
@misuzu try add to configuration:
```
  systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/tmp" ];
```

Reply:
@Izorkin this fixes my issue, thank you!